packages/ethers-5/src/utils/defaultEthersReactConfig.ts

```diff
@@ -0,0 +1,50 @@
+import '@web3modal/polyfills'
```

Comment:
We only needed to separate default config into core and react ones because there were slight type differences in `wagmi` and `wagmi/core` packages. Since you are using single ethers package, you just need one file with one `defaultConfig` definition.